backend/src/main/scala/com/softwaremill/adopttapir/template/scala/EndpointsView.scala

```diff
@@ -173,12 +182,30 @@ object EndpointsView {
 
   }
 
+  private object MetricsEndpoint {
+    def prepareMetricsEndpoint(serverEffect: ServerEffect): Code = {
+      val (effect, endpoint) = serverEffect match {
+        case ServerEffect.FutureEffect => ("Future", "ServerEndpoint[Any, Future]")
+        case ServerEffect.IOEffect => ("IO", "ServerEndpoint[Any, IO]")
+        case ServerEffect.ZIOEffect => ("Task", "ZServerEndpoint[Any, Any]")
+      }
+      Code(
+        s"""  val prometheusMetrics: PrometheusMetrics[${effect}] = PrometheusMetrics.default[${effect}]()
+           |  val metricsEndpoint: ${endpoint} = prometheusMetrics.metricsEndpoint""".stripMargin,
+        Set(
+          Import("sttp.tapir.server.metrics.prometheus.PrometheusMetrics")
+        )
```

Comment:
missing `Import("zio.Task")` which is detected when documentation is not requested

Reply:
Done